packages/internal/src/callback.js

```diff
@@ -315,7 +316,7 @@ harden(prepareAttenuator);
  */
 export const prepareGuardedAttenuator = (zone, interfaceGuard, opts = {}) => {
   // TODO: Handle symbolMethodGuards too?
-  const { methodGuards } = interfaceGuard;
+  const { methodGuards } = getInterfaceGuardPayload(interfaceGuard);
   const methodNames = ownKeys(methodGuards);
```

Comment:
As an aside, would you consider implementing this TODO as a drive-by (or at least enlighten me regarding the best way to get all the symbol and string method names for an `interfaceGuard`)?

Reply:
Drive by done.